server/asset/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/decred/dcrdex/asset
+module github.com/decred/dcrdex/server/asset
```

Comment:
I actually sniped this one in #39 too.

Reply:
Ah, cool.  Well let's get it in here so we can update the other mods.